boards/m5stack/m5stack_cores3/doc/index.rst

```diff
@@ -0,0 +1,117 @@
+.. _m5stack_cores3:
+
+M5Stack CoreS3
+##############
+
+Overview
+********
+
+M5Stack CoreS3 is an ESP32-based development board from M5Stack. It is the third generation of the M5Stack Core series.
+
+M5Stack CoreS3 features consist of:
+
+- ESP32-S3 chip (dual-core Xtensa LX7 processor @240MHz, WIFI, OTG and CDC functions)
+- PSRAM 8MB
+- Flash 16MB
+- LCD ISP 2", 320x240 pixel ILI9342C
+- Capacitive multi touch FT6336U
+- Camera 30W pixel GC0308
+- Speaker 1W AW88298
+- Dual Microphones ES7210 Audio decoder
+- RTC BM8563
+- USB-C
+- SD-Card slot
+- Geomagnetic sensor BMM150
+- Proximity sensor LTR-553ALS-WA
+- 6-Axis IMU BMI270
+- PMIC AXP2101
+- Battery 500mAh 3.7 V
+
+.. figure:: img/m5stack_cores3.webp
+        :align: center
+        :alt: M5Stack-CoreS3
+        :width: 400 px
+
+        M5Stack CoreS3 module
+
+Start Application Development
+*****************************
+
+Before powering up your M5Stack CoreS3, please make sure that the board is in good
+condition with no obvious signs of damage.
+
+System requirements
+===================
+
+Prerequisites
+-------------
+
+Espressif HAL requires WiFi and Bluetooth binary blobs in order work. Run the command
+below to retrieve those files.
+
+.. code-block:: console
+
+   west blobs fetch hal_espressif
+
+.. note::
+
+   It is recommended running the command above after :file:`west update`.
+
+Building & Flashing
+-------------------
+
+Build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: m5stack_cores3/esp32s3/procpu
+   :goals: build
+
+The usual ``flash`` target will work with the ``m5stack_cores3`` board
+configuration. Here is an example for the :zephyr:code-sample:`hello_world`
```

Comment:
board target, and needs to use the proper name as used with west build

Reply:
> board target, and needs to use the proper name as used with west build

Is this supposed to be board name? I saw  this in some other board docs. (like [this one](https://docs.zephyrproject.org/latest/boards/espressif/esp32s3_devkitm/doc/index.html))

Reply:
It is the board target, if you run `west build m5stack_cores3` you will get an error because there is no such board target